src/types.ts

```diff
@@ -811,6 +820,12 @@ export interface TokenWithBalance extends Token {
     ownershipLastUpdateTime: number;
 }
 
+export interface CollectionOwnership extends NFTCollection {
+    standard?: string;
+    blockchainDescriptor: string;
+    contractAddress?: string;
```

Comment:
is contractAddress really optional? having only blockchain does not say much about ownership

Reply:
We will have Solana in the future that doesn't have standard or address so we've putted it this way on nft-service in order not to be a breaking change

Reply:
I see. But no matter the blockchain, having just a descriptor does not say much, will there be other fields added?

Reply:
we have id, name, symbol and descriptor that are required for now, probably in the future there will be some other fields